### PR TITLE
Main added 2021-12-05-Polygon.md

### PR DESCRIPTION
**Pull request by Malyel**
Add article about Polygon

### CODE REVIEW

**Comment by Malyel** [comment]
Deleted ## Additional Details section as it is not included in standard sections

File: content/attacks/posts/2021-12-05-Polygon.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: "Polygon hacked for over 800,000 MATIC"
+---
+
+## Summary
+
+In early December 2021, Polygon, an Ethereum-based network has "silently fixed" a vulnerability that had put its native MATIC tokens worth $24 billion at risk. The issue came to light after a group of ethical hackers informed Immunefi, a bug bounty platform associated with decentralised finance (DeFi). Immunefi hosts the bug bounty for the Polygon network. Both white hat hackers who helped discover the bug were compensated a combined total of $3.46 million as part of the bug bounty program. However, before the recommended upgrade, other hacker(s) were able to steal 801,601 MATIC tokens, making up for around $2.4 million in terms for the date of the hack.
+
+## Attackers
+
+The perpetrators remain unidentified.
```
Comment (jhirschkorn) [inline content/attacks/posts/2021-12-05-Polygon.md:14]: add status of stolen funds

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**